### DISCUSSION TITLE
How to load trained weights and apply them for molecules with atoms not present in the training set?

### DISCUSSION THREAD

**Discussion by ziyueyang37** [ACEsuit/mace#208] [Announcements]
**Is your feature request related to a problem? Please describe.**
I want to access the MACE encoder's output in a set of molecules I'm interested into. I'll use this output in my workflow. However, I'm having trouble loading the weights for the MACE model and applying it to my molecules. 

**Describe the solution you'd like**
Is there a tutorial that I can follow to show me how to load and use your model?

**Additional context**
I am implementing MACE in my workflow and need to apply it for different molecules in my dataset. Is it a problem if my molecules have atom types that weren't present in your training set? My goal is to get the encoder's output and use it in my models. Is there an easy way of getting this done?

**Comment by ziyueyang37** [comment]
@maykcaldas

**Comment by davkovacs** [comment]
Hi! you can check tutorials here that should help. 
https://github.com/ilyes319/mace-tutorials
Also check the Documentation which describes how to evaluate the model:
https://mace-docs.readthedocs.io/en/latest/
